deepfly/CLI/main.py

```diff
@@ -0,0 +1,68 @@
+import math
+import argparse, os.path
+from pathlib import Path
+from deepfly.pose2d.drosophila import main as pose2d_main
+from deepfly.pose2d import ArgParse
+from ..GUI.Config import config
+from ..GUI.util.os_util import get_max_img_id
+
+def parse_cli_args():
+    parser = argparse.ArgumentParser(
+        description = "DeepFly3D pose estimation"
+    )
+
+    # Convention for faster vertical reading:
+    # - first line is argument, starting with short form (e.g. -f, --folder)
+    # - second line is help text (e.g. help="the input folder")
+    # - then whatever else is required.
+    # last line is closing parenthesis )
+
+    parser.add_argument(
+        "input_folder", 
+        help="Folder containing unlabeled images."
+    )
+    parser.add_argument(
+        "-n", "--num-images-max",
+        help="Maximal number of images to process.",
+        default=math.inf,
+        type=int
+    )
+    parser.add_argument( # TODO: there is "--stacks" in ArgParse, is it the same?
```

Comment:
It is the same thing with `--stacks`, we should use that one instead. you can remove this argument.